web/html/src/core/spa/spa-renderer.js

```diff
@@ -26,7 +26,10 @@ function hasReactApp() {
   return window.pageRenderers.spa.reactAppsName.length > 0;
 }
 
-function renderGlobalReact(element: ReactElement<any>, container: Element) {
+function renderGlobalReact(element: ReactElement<any>, container: ?Element) {
+  if (container == null) {
```

Comment:
Woah, sounds new to me. What's the scenario did you experienced it?

Reply:
We don't hit that case in real-life, but theoretically, this is usually called with `document.getElementById()`. `getElementById` can return `null` if the `id` does not exist and Flow knows about this. So basically, this is to satisfy Flow. I made the check in a single place here so we won't have to check wherever we call `getElementById`.

Reply:
Awesome, thanks for the explanation.